example/transition_vtol_fixed_wing/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(transition_vtol_fixed_wing)
+
+if(NOT MSVC)
+    add_definitions("-std=c++11 -Wall -Wextra -Werror")
+else()
+    add_definitions("-std=c++11 -WX -W2")
+    set(platform_libs "Ws2_32.lib")
+endif()
```

Comment:
Once you merge or rebase you will see that this file gets a lot simpler.

Reply:
The required changes are being documented in https://github.com/dronecore/docs/pull/62 (in progress). The interesting bit is [here](https://github.com/hamishwillee/docs-1/blob/4dcd412abc72b448e9260f81f5922ea776e5262f/en/guide/toolchain.md#dronecore-installed-system-wide-dronecore_installed_system) - basically we're changing to use system-wide install by default AND dronecore itself is now linking to the thread library - so you no longer need to do most of this work.